README.md

```diff
@@ -49,15 +50,20 @@ The default prefix is `.`. Commands are used through chat, use `.commands` for a
 ##### Bind modules
 Run `.bind <module> <key>`.
 
+You can also use `.bind modifiers on` to change binds to something like `ctrl + shift + w` or `ctrl + c`.
```

Comment:
> to change binds to something like 

This sort of implies that `.bind modifiers on` also assigns binds.
How about *You can also use `.bind modifiers on` to allow modules to be bound to keybinds with modifiers, e.g. `ctrl + shift + w` or `ctrl + c`*?